schema/catvrs/catvrs-source.yaml

```diff
@@ -13,296 +13,254 @@ namespaces:
-
-
-  NumberChange:
-    maturity: draft
-    # ga4ghDigest:
-    #   keys:
-    #     - numberChange
-    #   prefix: CX
+      A canonical allele is defined by an `Allele <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#variation>` 
+      that is representative of a collection of congruent Alleles, each of which depict the same nucleic acid 
+      change on different underlying reference sequences. Congruent representations of an Allele often exist
+      across different genome assemblies and associated cDNA transcript representations.
     type: object
-    description: >-
-      A quantitative assessment of a unit within a system (e.g. genome, cell,
-      etc.) relative to a baseline quantity.
     properties:
-      type:
-        type: string
-        const: "NumberChange"
-        default: "NumberChange"
-        description: >-
-          MUST be "NumberChange"
-      change:
-        oneOf:
-          - type: integer
-          - $refCurie: vrs:Range
-          - copyChange:
-            type: string
-            enum: [ "efo:0030069", "efo:0020073", "efo:0030068", "efo:0030067", "efo:0030064", "efo:0030070",
-                    "efo:0030071", "efo:0030072" ]
-        description: >-
-          a quantitative or qualitative value of the measurement with respect to a baseline (0). 
-          If qualitative, must be one of "efo:0030069" (complete genomic loss), "efo:0020073" (high-level loss),
-          "efo:0030068" (low-level loss), "efo:0030067" (loss), "efo:0030064" (regional base ploidy),
-          "efo:0030070" (gain), "efo:0030071" (low-level gain), "efo:0030072" (high-level gain).
-        required: [ "change" ]
+      # type:
+      #   extends: type
+      #   const: CanonicalAllele
+      #   default: CanonicalAllele
+      #   description: 'MUST be "CanonicalAllele"'
+      constraints:
+        INCLUDES: # I DON'T THINK THIS IS THE RIGHT METHOD, NOT SURE WHERE TO LOOK UP WHAT CORRECT METHOD IS?
+          definingContexConstraint
+    required:
+      - constraints
 
-  QuantityVariance:
+  CategoricalCNV:
     maturity: draft
-    type: object
-    # inherits: NumberCount
+    inherits: CategoricalVariant
     description: >-
-      The Quantity Variance class captures one axis of variation in the generalized model of categorical
-      variation.  It is used to model quantitative measure changes in a given biological level of
-      variation.
+      description of CNVs
+    type: object
     properties:
-      type:
-        type: string
-        const: QuantityVariance
-        default: QuantityVariance
-        description: 'MUST be "QuantityVariance"'
-      # label:
-  #       extends: label
+      # type:
+      #   extends: type
+      #   const: CanonicalAllele
+      #   default: CanonicalAllele
+      #   description: 'MUST be "CanonicalAllele"'
+      constraints:
+        INCLUDES: # I DON'T THINK THIS IS THE RIGHT METHOD, NOT SURE WHERE TO LOOK UP WHAT CORRECT METHOD IS?
+          copyConstraint
```

Comment:
you are extending an `array`, type should remain the same, and use items to define what objects fit into this array. Those objects SHOULD be the constraints you consider relevant to a profile.